.github/workflows/tests.yml

```
@@ -0,0 +1,71 @@
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+      - name: Run tests
+        run: |
+            git config --global --add safe.directory "$(pwd)"
+            git remote set-url origin "${{ github.server_url }}/${{ github.repository }}"
+            git clone --branch "v$Env:CMLIB_VERSION" https://github.com/cmakelib/cmakelib.git
+            $Env:CMLIB_DIR=$(Join-Path -Path $(Get-Location).Path -ChildPath "cmakelib")
+            cd test/ && cmake .
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)